### ISSUE TITLE
Could not reproduce the reported performance on eth data split

### ISSUE THREAD

**Issue by azhugreat** [cocoon2wong/SocialCirclePlus#4]
Thank you for your contribution to the community. Recently when I try to reproduce the reported performance of v2Net and social circle on eth-ucy dataset and SDD dataset with this repository, the ADE and FDE of eth split is abnormally high, about 0.61/1.28 instead of  0.25/0.37 in the paper. Other split trained with the same parameter have the same result in the paper. 

Maybe there is some parameter setting or data preprocess mistake in my process or the codebase, much obliged for your help.

**Comment by cocoon2wong** [comment]
Hello again @azhugreat !

Thanks for your questions!
Could you please provide us your command to start training this model?

Since we have not provide our PyTorch weights of this `vsc` model, I have tried to test the `vscp` model again with command:
```bash
python main.py -sc ../Project-Monandaeg-SC+/vspceth_adaptive
```
And it seems working fine with the codes in this repo.

You can tried to training a new `vsc` model on this split by simply using
```
python main.py --restore_args ../Project-Monandaeg-SC+/vspceth_adaptive --model vsc
```
to reproduce our training process. (You can find this weight from https://github.com/cocoon2wong/Project-Monandaeg/tree/SocialCirclePlus. Also, this `SocialCirclePlus` repo works with those wights published in the `SocialCircle` repo.)

I'm sorry I can't start re-training right away due to computing device limitations. My guess is that a parameter was set incorrectly. I look forward to hearing from you!

Wong

**Comment by azhugreat** [comment]
Thank you for your reply! I have downloaded the provided weights and reproduced the report result. Thank you for your response and help! 

But I still cannot figure out the problem within the training process of eth split. I have re-downloaded this repo and tried to train eth split **without any modification**, but there is still problem in eth split. The training command is :
`python main.py --model vspc --split eth --partitions 8`
then the training result is:
`Training...: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████| 500.00/500.00 [22:45<00:00,  2.73s/it, epoch=499, best=[0.61268127 1.214137  ], l2(Loss)=0.13211793, ADE(Metrics)=0.7282983, FDE(Metrics)=1.4920024]`

If I tried to reproduce the training process by command:
`python main.py --restore_args ./Project-Monandaeg-SocialCirclePlus/Project-Monandaeg-SocialCirclePlus/vspceth_hard --model vspc`
then the training process is:
`Training...:   1%|▊                                                                          | 2.00/200.00 [00:21<36:08, 10.95s/it, epoch=1, best=[1.9848926 2.7671084], l2(Loss)(keypoints)=0.78917694, ADE(Metrics)(keypoints)=1.9848926, FDE(Metrics)(keypoints)=2.7671084]`

Is there any pre-set parameters related to only the eth set but not other sets? Thank you for your help!

**Comment by cocoon2wong** [comment]
Hi @azhugreat,

Some of the default args (args that was not provided in the form of `--ARG_NAME` when training) should be assigned, such as `--lr` (learning rate), `--batch_size`, `--epochs`. Changing these args may heavily affect the training results, which applies to all datasets and splits.

As for the `eth` split, since the test clip `eth` has different trajectory distributions to other `ETH-UCY` sets (like mentioned by other researchers), the only different arg we used is the `--preprocess`. It has been set to `101` on `eth`, indicating that the `MOVE` and `ROTATE` preprocesses will be made before implementing, and it will be set  to `100` on all other sets that the `MOVE` preprocess will be made. You can compare their `args.json` files to better learn their differences, like the settings in `eth` in https://github.com/cocoon2wong/Project-Monandaeg/blob/SocialCirclePlus/vspceth_adaptive/args.json, and those in `zara1` in https://github.com/cocoon2wong/Project-Monandaeg/blob/SocialCirclePlus/vspczara1_adaptive/args.json.

I also noticed that your provided training logs with the `--restore` arg only runs for the very first epoch. Does it run correctly in the following epochs?

**Comment by azhugreat** [comment]
Thank you for your help! After changing the --preprocess to 101 at  eth split, I could train the model correctly and fix all my problems. Sorry for my ignorance of the provided args.json .

Much obliged, Yinglong Zhu
